### PR TITLE
git-reset: clarify patch example and use long option

### PR DESCRIPTION
**Pull request by marchersimon**

### CODE REVIEW

**Review by CleanMachine1 — APPROVED** [review]

**Comment by bl-ue** [comment]
This is great, but there are three translations that need updating.  Specifically:
- [Spanish](https://github.com/tldr-pages/tldr/blob/main/pages.es/common/git-reset.md) - @navarroaxel?
- [French](https://github.com/tldr-pages/tldr/blob/main/pages.fr/common/git-reset.md) - @nicokosi / @patricedenis?
- [Italian](https://github.com/tldr-pages/tldr/blob/main/pages.it/common/git-reset.md) - @glenacota / @mebeim?

@marchersimon at least can you expand `-p` in the translations?

**Review by bl-ue — CHANGES_REQUESTED** [review]
This is great, but there are three translations that need updating.  Specifically:

- [Spanish](https://github.com/tldr-pages/tldr/blob/main/pages.es/common/git-reset.md) - @navarroaxel?
- [French](https://github.com/tldr-pages/tldr/blob/main/pages.fr/common/git-reset.md) - @nicokosi / @patricedenis?
- [Italian](https://github.com/tldr-pages/tldr/blob/main/pages.it/common/git-reset.md) - @glenacota / @mebeim?

@marchersimon at least can you expand `-p` in the translations?

**Comment by nicokosi** [comment]
My suggestion for [French](https://github.com/tldr-pages/tldr/blob/main/pages.fr/common/git-reset.md) :

> Enlever des fichiers spécifiques de la *zone de stage* avec le mode interactif  :

File: pages.fr/common/git-reset.md [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 - Enlever une portion d'un fichier de la *zone de stage* :
```
Comment (patricedenis) [inline pages.fr/common/git-reset.md:16]:
```suggestion
- Enlever, en mode interactif, des fichiers spécifiques de l’index :
```

**Review by patricedenis — COMMENTED** [review]
I wil change all the references in French from *zone de stage* to "l’index" because this is the real French term for "the stage area"

File: pages.fr/common/git-reset.md (hunk 1 above)
Comment (patricedenis) [reply pages.fr/common/git-reset.md:16]: could you do that also @marchersimon ?

File: pages.es/common/git-reset.md [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
 - Desmarca partes de un archivo:
```
Comment (navarroaxel) [inline pages.es/common/git-reset.md:15]:
```suggestion
- Interactivamente desmarca partes de un archivo:
```

**Review by navarroaxel — APPROVED** [review]

File: pages.it/common/git-reset.md [hunk 3]
```diff
@@ -14,7 +14,7 @@
 
 - Rimuovi dall'area di stage solo alcune porzioni di un file:
```
Comment (marchersimon) [inline pages.it/common/git-reset.md:15]:
```suggestion
- Rimuovi dall'area di stage solo alcune porzioni di un file in modo interattivo:
```
From the other italian pages it looks like the most common translation of `interactively` is just appending `in modo interattivo` to the sentence.

**Review by patricedenis — APPROVED** [review]

**Review by glenacota — APPROVED** [review]
LGTM, nice one!!

File: pages.it/common/git-reset.md (hunk 3 above)
Comment (bl-ue) [reply pages.it/common/git-reset.md:15]: Go for it, @glenacota 👍🏻'd it.

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @marchersimon!

**Comment by sbrl** [comment]
@bl-ue, are all translations now updated?

**Review by bl-ue — APPROVED** [review]
Yep, they are.  We've got a big load of approvals so it's definitely okay to merge. 🚂

**Pull request merged**